experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -0,0 +1,17 @@
+{% extends "experimenter_base.html" %}
+
+{% block title %}
+  History
+{% endblock %}
+
+{% block content %}
+    <h1>Changes Overview</h1>
+    <p>Experiment Slug: {{ object.slug }}</p>
+    {% for changelog in object.get_changelogs %}
```

Comment:
Just use `experiment.changelogs.all` -- the list optimization I suggested was premature.

Reply:
I assume we won't remove the  `get_changelogs` method inside the model as it will be required for pre processing, I'll just return the changelogs directly for now without using the `list` method